huaweicloud/services/acceptance/cpts/resource_huaweicloud_cpts_task_test.go

```diff
@@ -0,0 +1,94 @@
+package cpts
+
+import (
+	"fmt"
+	"strconv"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+
+	"github.com/huaweicloud/huaweicloud-sdk-go-v3/services/cpts/v1/model"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance"
+)
+
+func getTaskResourceFunc(conf *config.Config, state *terraform.ResourceState) (interface{}, error) {
+	client, err := conf.HcCptsV1Client(acceptance.HW_REGION_NAME)
+	if err != nil {
+		return nil, fmt.Errorf("error creating CPTS v1 client: %s", err)
+	}
+
+	id, err := strconv.ParseInt(state.Primary.ID, 10, 32)
+	if err != nil {
+		return nil, fmt.Errorf("the Task ID must be integer: %s", err)
+	}
+
+	request := &model.ShowTaskRequest{
+		TaskId: int32(id),
+	}
+
+	return client.ShowTask(request)
+}
+
+func TestAccTask_basic(t *testing.T) {
+	var obj model.CreateTaskResponse
+
+	rName := acceptance.RandomAccResourceName()
+	updateName := acceptance.RandomAccResourceName()
+	resourceName := "huaweicloud_cpts_task.test"
+
+	rc := acceptance.InitResourceCheck(
+		resourceName,
+		&obj,
+		getTaskResourceFunc,
+	)
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:          func() { acceptance.TestAccPreCheck(t) },
+		ProviderFactories: acceptance.TestAccProviderFactories,
+		CheckDestroy:      rc.CheckResourceDestroy(),
+		Steps: []resource.TestStep{
+			{
+				Config: testTask_basic(rName, 100),
```

Comment:
how about using another value for `benchmark_concurrency` as the default value is `100`.

Reply:
ok